src/AppBundle/Entity/Report/Report.php

```diff
@@ -834,6 +843,24 @@ public function setAgreedBehalfDeputyExplanation($agreedBehalfDeputyExplanation)
         return $this;
     }
 
+
+    /**
+     * @return Document[]
```

Comment:
the client only use arrays, arrayCollection is part of doctrine libs. It could potentially be introduced if needed, but not sure JMS will be able to hydrate using it